docs/howto/operator.rst

```diff
@@ -1170,6 +1172,8 @@ it will be retrieved from the GCP connection used. Both variants are shown:
     :start-after: [START howto_operator_cloudsql_delete]
     :end-before: [END howto_operator_cloudsql_delete]
 
+Note: If the instance has read or failover replicas you need to delete them first.
```

Comment:
I'd say "you need to delete them before you delete the primary instance.".

Reply:
Sure, I will change it.

Reply:
Done.